telescope/src/main/java/com/mattprecious/telescope/TelescopeLayout.java

```diff
@@ -335,34 +394,60 @@ private void trigger() {
       vibrator.vibrate(VIBRATION_DURATION_MS);
     }
 
+    switch (screenshotMode) {
+      case SYSTEM:
+        if (projectionManager != null && !screenshotChildrenOnly && screenshotTarget == this) {
+          // Take a full screenshot of the device. Request permission first.
+          registerRequestCaptureReceiver();
+          getContext().startActivity(new Intent(getContext(), RequestCaptureActivity.class));
+          break;
+        }
+
+        // System was requested but isn't supported. Fall through.
+      case CANVAS:
+        captureCanvasScreenshot();
+        break;
+      case NONE:
+        new SaveScreenshotTask(null).execute();
+        break;
+      default:
+        throw new IllegalStateException("Unknown screenshot mode: " + screenshotMode);
+    }
+  }
+
+  private void captureCanvasScreenshot() {
+    capturingStart();
+
+    // Wait for the next frame to be sure our progress bars are hidden.
+    post(new Runnable() {
```

Comment:
Had a test screenshot today with the progress bars included. Posted all the work to the next frame to make sure the bars are hidden.